pkg/config/controller_config.go

```diff
@@ -18,6 +18,7 @@ const (
 	flagDefaultTags                                  = "default-tags"
 	flagDefaultTargetType                            = "default-target-type"
 	flagExternalManagedTags                          = "external-managed-tags"
+	flagServiceTargetENISGTags                       = "service-target-eni-security-group-tags"
```

Comment:
how about just `endpoint-eni-security-group-tags`

Reply:
The flag is specific to Service, so the "service-" prefix is appropriate.